content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html

```diff
@@ -26,7 +26,10 @@
         <div class="row">
 
      <div class="col-md-8">
+          <!--
           <h3>Objectives</h3>
+          -->
+          <h3>目的</h3>
```

Comment:
```suggestion
<h3>目标</h3>
```
其他类似的地方（例如https://kubernetes.io/zh/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/）使用的是“目标”，尽量保持一致。谢谢你的第一个贡献。

Reply:
好的，感谢您的回复。